File: src/main/java/seremis/geninfusion/soul/gene/newAI/eatGrass/GeneAIEatGrass.java
package seremis.geninfusion.soul.gene.newAI.eatGrass;

import seremis.geninfusion.api.soul.IAllele;
import seremis.geninfusion.api.soul.IGene;
import seremis.geninfusion.soul.allele.AlleleBoolean;

public class GeneAIEatGrass implements IGene {
    @Override
    public Class<? extends IAllele> possibleAlleles() {
        return AlleleBoolean.class;
    }
}

File: src/main/java/seremis/geninfusion/soul/gene/newAI/GeneAIEatGrass.scala
package seremis.geninfusion.soul.gene.newAI

import seremis.geninfusion.api.soul.lib.Genes
import seremis.geninfusion.soul.MasterGene

class GeneAIEatGrass extends MasterGene(classOf[Boolean]) {
    addControlledGene(Genes.GENE_AI_EAT_GRASS_INDEX)
}
